<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>305</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(211, 215, 207);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>JetBrains Mono</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <property name="font">
       <font>
        <family>JetBrains Mono</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <attribute name="title">
       <string>QRZ</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>JetBrains Mono</family>
           <pointsize>11</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>QRZ Name:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="qrzname_field">
         <property name="font">
          <font>
           <family>JetBrains Mono</family>
           <pointsize>11</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>W1AW</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>JetBrains Mono</family>
           <pointsize>11</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>QRZ Password:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="qrzpass_field">
         <property name="font">
          <font>
           <family>JetBrains Mono</family>
           <pointsize>11</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Secret</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>JetBrains Mono</family>
           <pointsize>11</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>QRZ URL:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="qrzurl_field">
         <property name="font">
          <font>
           <family>JetBrains Mono</family>
           <pointsize>11</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>http://xmldata.qrz.com/xml/</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QWidget" name="widget_3" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QCheckBox" name="usehamdb_checkBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 1px solid red;</string>
            </property>
            <property name="text">
             <string>Use HamDB</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="useqrz_checkBox">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
              <pointsize>11</pointsize>
              <bold>false</bold>
             </font>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 1px solid red;</string>
            </property>
            <property name="text">
             <string>Use QRZ</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>CAT</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>JetBrains Mono</family>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>Rig Control</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>JetBrains Mono</family>
           <pointsize>11</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Host:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="rigcontrolip_field">
         <property name="font">
          <font>
           <family>JetBrains Mono</family>
           <pointsize>11</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>127.0.0.1</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <family>JetBrains Mono</family>
           <pointsize>11</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Port:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="rigcontrolport_field">
         <property name="font">
          <font>
           <family>JetBrains Mono</family>
           <pointsize>11</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>4532</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="radioButton_flrig">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
             </font>
            </property>
            <property name="toolTip">
             <string extracomment="Connect to FLRIG XMLRPC Server"/>
            </property>
            <property name="text">
             <string>FLRIG</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_rigctld">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
             </font>
            </property>
            <property name="toolTip">
             <string extracomment="Connect to rigctld daemon"/>
            </property>
            <property name="text">
             <string>rigcltd</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_none">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
             </font>
            </property>
            <property name="toolTip">
             <string extracomment="Use no rig control"/>
            </property>
            <property name="text">
             <string>None</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>CW</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <family>JetBrains Mono</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CW_Address:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="cwip_field">
         <property name="font">
          <font>
           <family>JetBrains Mono</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>localhost</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <family>JetBrains Mono</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CW_Port:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="cwport_field">
         <property name="font">
          <font>
           <family>JetBrains Mono</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>6789</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QWidget" name="widget_2" native="true">
         <property name="minimumSize">
          <size>
           <width>31</width>
           <height>22</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QRadioButton" name="usecwdaemon_radioButton">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>cwdaemon</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="usepywinkeyer_radioButton">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>PyWinkeyer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_5">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>None</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>JetBrains Mono</family>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
